addons/sourcemod/scripting/mapchooser_extended.sp

```diff
@@ -2465,16 +2465,16 @@ public int Native_GetMapAdminRestriction(Handle plugin, int numParams)
 	GetNativeString(1, map, len+1);
 
 	// Check if client should bypass admin restrictions
-	if(client < 0 || client > MaxClients)
-		return false;
-
-	// Client has bypass flag, dont return admin restrictions
-	if(CheckCommandAccess(client, "sm_nominate_ignore", ADMFLAG_GENERIC))
-		return false;
+	if(client >= 1 && client <= MaxClients)
```

Comment:
why do we even have to make this check here, cant we just return the value from the config file?

Reply:
The return as false mean don't apply the restriction to the client for nominate the map.

Reply:
oh okay well then the function name is really confusing, it should be something like `IsClientMapAdminRestricted`